graspy/simulations/simulations.py

```diff
@@ -695,7 +695,7 @@ def rdpg(X, Y=None, rescale=False, directed=False, loops=False, wt=1, wtargs=Non
            [0., 0., 0., 1., 1.],
            [0., 1., 1., 0., 0.],
            [1., 1., 1., 0., 0.]])
-
+           
```

Comment:
@dfrancisco1998 what is the point of this change? just making sure it doesnt mess up doctest or anything

Reply:
This is the correct doctest (same one from master). Sorry I was trying to correct the edits I had made in the file in my branch and must have added in an extra space.

Reply:
mind removing this change then?